lib/msf/core/exploit/remote/http_server.rb

```diff
@@ -172,19 +158,10 @@ def start_service(opts = {})
       print_status("Intentionally using insecure SSL compression. Your operating system might not respect this!")
     end
 
+    print_status("Using URL: #{proto}://#{srvhost_addr}#{uopts['Path']}")
```

Comment:
Looks like the port was dropped here, was that intentional? 👀 
```
[*] Using URL: http://192.168.123.6/R3ieTAhOv
```

Reply:
You're right, that's a mistake. I was intending to leave the port out when it's the default based on the schema like `#get_uri` does and I got mixed up. Thanks for catching this, fixed in 6e1d369ad308bd0dcbb20cfefbf1cc91c0b4bd28.

Reply:
I think this needs fixed still 👀 


My lport is 8080 for the web delivery and the port is missing:
```
[*] Started reverse TCP handler on private_ip:4444 
[*] Using URL: http://public_ip/iXjtZTZH7
```

Reply:
Totally unrelated to this PR, I wish there was an easy way to see all the paths that were registered with the http service manager, or at least see where the web delivery module is running from. It's pretty easy to forget which paths are for what after the console screen's been cleared, and it's not possible to grab the info from the `jobs` or service manager details from what I can see

```
msf6 exploit(multi/script/web_delivery) > jobs

Jobs
====

  Id  Name                                Payload                         Payload opts
  --  ----                                -------                         ------------
  2   Exploit: multi/script/web_delivery  python/meterpreter/reverse_tcp  tcp://x.x.x.x:4444
```

```
msf6 exploit(multi/script/web_delivery) > _servicemanager 
Services
========

 Id  Name                                                                References
 --  ----                                                                ----------
 0   HTTP Server                                                         2
 1   Rex::Proto::Http::Server8080-x.x.x.x-Rex::Socket::Comm::Local       2
```

Reply:
Yeah it looks like when I rebased, that change was lost. Let me do it again. Re-added in commit ff0ecfa49ef8d4b7c0d2ada0e00d17aa31747afe.